tests/unit_tests/physics/test_isotope_equilibrium_fractionation_factors.py

```diff
@@ -0,0 +1,103 @@
+# pylint: disable=missing-module-docstring
+import numpy as np
+import pint
+import pytest
+
+from PySDM import Formulae
+from PySDM.physics import constants_defaults
+from PySDM.physics.dimensional_analysis import DimensionalAnalysis
+
+CASES = (
+    ("HDO", "ice", {-120: 1.82, 0: 1.13}),
```

Comment:
add units!

Reply:
addressed in https://github.com/open-atmos/PySDM/pull/1063/commits/a778724dece518687b7a6e98d784a89aa182a077